UnityProject/Assets/Scripts/Items/IDCard.cs

```diff
@@ -215,26 +216,33 @@ public void SyncIDCardType(IDCardType oldCardType, IDCardType cardType)
 		pickupable.RefreshUISlotImage();
 	}
 
+	public IEnumerable<Clearance> GetClearance()
```

Comment:
a get method in my beautiful C#?! This is not java! Use properties instead.

Reply:
![image](https://user-images.githubusercontent.com/56727168/183138014-b02f12a0-cc78-4777-9df8-f679a2fdbf47.png)

Hummmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm

Reply:
I was young and stupid back then

Reply:
aaaaaah no, it was because it is an interface. We couldn't do properties in interfaces in that unity version. I wasn't so stupid, hehe. It's been long, I barely remember how I set up the whole thing so my review can only be superficial and hope for the best.